sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -75,6 +69,31 @@
       }
     } else if (id == R.id.action_web_map) {
       airMapInterface = mapViewBuilder.builder(AirMapViewTypes.WEB).build();
+    } else if (id == R.id.action_web_map_no_touch) {
+      airMapInterface = mapViewBuilder.builder(AirMapViewTypes.WEB).build();
+      final AirMapInterface finalAirMapInterface = airMapInterface;
+      airMapInterface.setOnMapLoadedListener(new OnMapLoadedListener() {
+        @Override
+        public void onMapLoaded() {
+          finalAirMapInterface.setTouchEnabled(false);
+        }
+      });
+    } else if (id == R.id.action_native_map_no_touch) {
+      try {
+        airMapInterface = mapViewBuilder.builder(AirMapViewTypes.NATIVE).build();
+        final AirMapInterface finalAirMapInterface1 = airMapInterface;
+        airMapInterface.setOnMapLoadedListener(new OnMapLoadedListener() {
+          @Override
+          public void onMapLoaded() {
+            finalAirMapInterface1.setTouchEnabled(false);
+          }
+        });
```

Comment:
@petzel in the GoogleMapOptions there isn't the same method ```setAllGesturesEnabled``` but there are ```tiltGesturesEnabled, rotateGesturesEnabled, zoomGesturesEnabled, scrollGesturesEnabled```.

Do you prefer if we use the ```setAllGuesturesEnabled``` method, which means we have to set it on a OnMapLoadedListener or would you want me to set all of the above 4 methods?